### PR TITLE
update squish to 7.1 with use qt6.4

### PR DESCRIPTION
**Pull request by saw-jan**
Updated Squish to 7.1.1 with Qt6.4 and Python to 3.10 required by Squish.
Depends on https://github.com/owncloud-ci/client/pull/27

Tested: https://drone.owncloud.com/owncloud/client/16291 :heavy_check_mark:

### CODE REVIEW

**Comment by saw-jan** [comment]
@fmoc Could you review? The PR is ready.

File: .drone.star [hunk 1]
```diff
@@ -25,7 +25,7 @@ def main(ctx):
     'repo': ctx.repo.name,
     'squishversion': {
         'latest': '6.7-20220106-1008-qt515x-linux64',
-        'fedora': '6.7-20220106-1008-qt515x-linux64',
+        'fedora': '7.1.1-qt64x-linux64',
         'qt512': '6.7-20210421-1504-qt512x-linux64',
```
Comment (individual-it) [inline .drone.star:29]: should we have an other tag `qt64` or is that not needed?

File: .drone.star [hunk 2]
```diff
@@ -25,7 +25,7 @@ def main(ctx):
     'repo': ctx.repo.name,
     'squishversion': {
         'latest': '6.7-20220106-1008-qt515x-linux64',
```
Comment (individual-it) [inline .drone.star:27]: latest should use 7.1.1

**Review by fmoc — COMMENTED** [review]
Looks good overall. @individual-it has a few good points which I also was about to comment on. Will re-review once fixed.

File: .drone.star (hunk 1 above)
Comment (saw-jan) [reply .drone.star:29]: With this, these tags will  be created/updated
```bash
owncloudci/squish:fedora
owncloudci/squish:fedora-7.1.1-qt64x-linux64
owncloudci/squish:fedora-38-7.1.1-qt64x-linux64
```

**Review by fmoc — APPROVED** [review]

**Pull request merged**